domains/yusuf.json

```diff
@@ -0,0 +1,13 @@
+{
+  "description": "My Github pages portfolio",
+  "repo": "https://github.com/YAOfficial/portfolio/tree/gh-pages",
+  "owner": {
+    "username": "YAOfficial",
+    "email": "yahumble1@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://yaofficial.github.io/portfolio/",
```

Comment:
This is an invalid cname entry. CNAME records have to be a host. Something like `yaofficial.github.io`.
I'd suggest renaming the repo name to `yaofficial.github.io`. You can take a look at the [github pages guide](https://guides.github.com/features/pages/)

Reply:
oh i was unaware will fix asap

Reply:
I've updated it for you. Should be good to go. :)

Reply:
oh so sorry to bother you dude but my repo has -dahes instead of . dots https://github.com/YAOfficial/yaofficial-github-io  i think thats  why it isnt working for me currently

Reply:
im rly sry to  bother you dude but im getting this error when i set the custom domain name in github pages "Domain's DNS record could not be retrieved. " here is the repo https://github.com/YAOfficial/yaofficial.github.io/ and i set the custom domain to yusuf.is-a.dev i hope i did everything correctly

Reply:
The domain will take some time to reflect. Create a support issue if this issue persists after a day.